### DISCUSSION TITLE
Backups smaller than real volume usage. Partial backup ?

### DISCUSSION THREAD

**Discussion by Willena** [longhorn/longhorn#3536] [Q&A]
I'm trying to setup automatic backups for my PV/PVC. 
I store backups in an S3 bucket from Scaleway. It seems to be partially working as longhorn uploaded some objects (the bucket is dedicated to backups made by longhorn) 

For now, I'm trying with two PVC, one with a size of 2G and 150M of data in it and another PVC of 1G with 50M in it. 
I created a recurring backup job every day at night for these volumes (concurrency 1, keep 7).

After a week, the content of the object store is quite strange. It only weights a total of 16 MB (that volume should include backups for the two volumes described before). Backup / snapshots are incremental, so shouldn't have I at least have 200M in the bucket ?

Any suggestions on what can be configured wrongly ? It looks like the first chunk of data never got sent and it only backed up the increments of data.

**Comment by derekbit** [comment]
Backup job compresses the data, so the size may be smaller. If you're worried, you can probably restore them back and check your data.

**Reply by Willena** [reply]
Oh ! I now understand a little bit better. 
What algorithm is used for the compression ? 
Also I just saw that the backup config file contains the following part:
```
  "SingleFile": {
    "FilePath": ""
  }
```
How can I enable single file backup ?

**Reply by derekbit** [reply]
> What algorithm is used for the compression ?

Longhorn compresses the volume backup data by gzip currently.
